src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final int EMPTY_ARRAY = 0;
+    private static final String SEPARATOR = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readFromFile(fromFileName)));
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        List<String> list;
+        try {
+            list = Files.readAllLines(Paths.get(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read the file: " + fromFileName, e);
+        }
+        return list.toArray(new String[EMPTY_ARRAY]);
+    }
+
+    private StringBuilder createReport(String[] fromFileArray) {
```

Comment:
```suggestion
    private String createReport(String[] fileRows) {
```

I think It's better to return String from this method